### ISSUE TITLE
Lexical error when opening a document

### ISSUE THREAD

**Issue by viktomas** [jonathanyeung/mark-sharp#9]
**Describe the bug**
I try to open a document in mark sharp and I get an empty webview and an error

```
Webview Error: Minified Lexical error #66; visit https://lexical.dev/docs/error?code=66&v=7 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
```

**To Reproduce**

Put the following txt into a text editor

```
- [[2024-05-15]] [Slash Commands Not Disabled When Duo Features are Disabled for a Project](https://gitlab.com/gitlab-org/gitlab-vscode-extension/-/issues/1360#note_1906444540)
```

**Expected behavior**
Shows the same rendered markdown as VS Code preview

**Screenshots**

![image](https://github.com/jonathanyeung/mark-sharp/assets/1079718/bb505bab-6923-45e9-811b-19c63e6eae7b)

**Platform (please complete the following information):**
 - OS:  MacOs 14.5 (23F79)
 - Extension Version `1.2.2`

VS Code version:

```
Version: 1.90.0-insider
Commit: 4207ee1f8ddac59df509b75e4bdd1c77904f766e
Date: 2024-05-22T02:04:29.708Z
Electron: 29.3.1
ElectronBuildId: 9464424
Chromium: 122.0.6261.156
Node.js: 20.9.0
V8: 12.2.281.27-electron.0
OS: Darwin arm64 23.5.0
```

**Comment by jonathanyeung** [comment]
Thanks for reporting this issue! I can reproduce it - I'll have a fix for it in the next patch version.

**Comment by jonathanyeung** [comment]
Hi @viktomas - this should be fixed in the latest version, `1.2.3` - let me know if you're still encountering issues, thanks for reporting this issue!

**Comment by viktomas** [comment]
Thank you @jonathanyeung. The issue seems to be fixed, I'll let you know if it reappears
